assets/xml/objects/object_kin2_obj.xml

```diff
@@ -1,34 +1,34 @@
 ﻿<Root>
     <File Name="object_kin2_obj" Segment="6">
         <DList Name="object_kin2_obj_DL_000120" Offset="0x120" />
-        <DList Name="object_kin2_obj_DL_000128" Offset="0x128" />
-        <DList Name="object_kin2_obj_DL_0002C0" Offset="0x2C0" />
-        <DList Name="object_kin2_obj_DL_000360" Offset="0x360" />
-        <Collision Name="object_kin2_obj_Colheader_000490" Offset="0x490" />
+        <DList Name="gOceanSpiderHouseDebrisDL" Offset="0x128" />
```

Comment:
Is this general debris, or just in the bombable wall? If so it should get the same BombableWall prefix.

Reply:
This is just for the bombable wall. Will be in next pushed commit.